server/middleware/validation/signinValidation.js

```diff
@@ -0,0 +1,25 @@
+const Joi = require('joi');
+const boomify = require('../../utilis/boomify');
+
+const signinValidation = async (req, res, next) => {
+  const { email, password } = req.body;
+  const schema = Joi.object({
+    email: Joi.string().email().required(),
+    password: Joi.string().min(6).alphanum().required()
+      .messages({
+        'string.min': 'password should be at least 8 characters',
+        'any.required': 'password is a required field',
+      }),
+  });
+  try {
+    await schema.validateAsync(
+      { email, password },
+      { abortEarly: false },
+
+    );
+    next();
+  } catch (error) {
+    next(boomify(400, error.details.map((e) => e.message).join('\n')));
+  }
+};
+module.exports = signinValidation;
```

Comment:
you are doing create job here, but I need you to stick to your teammate file structure and make validation in the utils folder, not as middleware.